release-notes/8.0/8.0.0/api-diff/Microsoft.NETCore.App/8.0.0_System.Numerics.md

```diff
@@ -0,0 +1,247 @@
+# System.Numerics
```

Comment:
System.Numerics LGTM

This is namely support for `IUtf8SpanFormattable`/`IUtf8SpanParsable`, an iterative Crc32C primitive, `Lerp`, `DegreesToRadians`, `RadiansToDegrees`, left-handed matrix creation functions, and removing the `where T : struct` contstraint from `Vector<T>` so it can more easily be used in generic contexts (this does not expand the set of supported `T`).

CC. @dotnet/area-system-numerics